packages/channels/botpress-channel-rocketchat/src/incoming.js

```diff
@@ -0,0 +1,27 @@
+module.exports = (bp, rocketchat) => {
+  const isDirect = messageType => {
+    if(messageType == 'd') return true; else return false;
```

Comment:
Could you install and configure ESLint + Prettier so that the code style is like the other channels (for consistency purposes) ?